.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!--- Provide a general summary of your changes in the title above -->
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (a non-breaking change which fixes an issue)
+- [ ] New feature (a non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+
+
+## Description
+<!--- Describe your changes in detail -->
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it resolves an open issue, please link to the issue here. For example "Resolves: #137" -->
```

Comment:
Is there anything to do to enable links between Issues and PRs?
I mean, I'm talking about "Resolves" phrase.

Reply:
yes, 'resolves' is a magic keyword that github uses to link pr -> issue